### PR TITLE
Add vic-machine version

### LINKED ISSUE #934: access to vic-machine version number
Users need a way to determine the version of vic-machine they are running. Whereas we have  <code>docker info</code> and <code>docker version</code> commands to get the version info once a VCH has been deployed, we need a way to get the version info when customers seek support due to inability to deploy a VCH. One possible solution is to implement a <code>vic-machine version</code> command.

### PR DESCRIPTION
**Pull request by emlin**
Fixes #934 

  show the version as <build number>-<commit sha>

Sample: 
vic-machine-linux version 2254-7bd1af210b87615797163437ac4ec9de622aba4b

### CODE REVIEW

File: cmd/vic-machine/version/version.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```
Comment (caglar10ur) [inline cmd/vic-machine/version/version.go]: Did you commit this file by mistake?

Comment (emlin) [inline cmd/vic-machine/version/version.go]: You got it. Thanks

**Comment by andrewtchin** [comment]
lgtm
this is the output when built without drone:
```vic-machine-linux version -
```

**Comment by emlin** [comment]
@hickeng @mdubya66, are you OK with this version format?

**Comment by mdubya66** [comment]
can i see the output?

**Comment by andrewtchin** [comment]
```vic-machine-linux version 2254-7bd1af210b87615797163437ac4ec9de622aba4b```
2254 is drone build number, followed by commit hash

**Comment by mdubya66** [comment]
lgtm for now. we will need to re-visit this at some point

File: cmd/vic-machine/main.go [hunk 2]
```diff
@@ -62,9 +63,23 @@ func main() {
 			Action: inspect.Run,
 			Flags:  inspect.Flags(),
 		},
+		{
+			Name:   "version",
+			Usage:  "Show VIC version information",
+			Action: showVersion,
+		},
+	}
+	if Version != "" {
```
Comment (hmahmood) [inline cmd/vic-machine/main.go]: Why this check?

Comment (emlin) [inline cmd/vic-machine/main.go]: We need release version, but at this time, we cannot get that value, for we don't build after the build version is set.
While we have release version, e.g. tag set, to trigger build, this version can be set, and then we can get version like <release version>-<build number>-<commit sha>

**Comment by emlin** [comment]
♻️

**Pull request merged**